virtualizarr/xarray.py

```diff
@@ -99,6 +100,10 @@ def open_virtual_dataset(
         return open_virtual_dataset_from_v3_store(
             storepath=filepath, drop_variables=drop_variables, indexes=indexes
         )
+    if filetype == "dmr++":
+        with open(filepath, "rb") as f:
```

Comment:
The importing of `DMRParser` should happen inside the if statement here, so that the `dmr++`-related libraries can be optional dependencies.